src/routes/frontdesk/+page.ts

```diff
@@ -0,0 +1,14 @@
+import { pb } from "$lib/stores/authStore";
+
+export const load = async ({ fetch }) => {
+  // Sorting by relations does not work: https://github.com/pocketbase/pocketbase/discussions/1429
```

Comment:
Jeg tenkte litt på dette. Det fins tre muligheter, sortert etter hva jeg tror er best:


1. Problemet løses på backend, via relation sorting eller hva man vil. 
2. Problemet kan løses ved å flytte hele datamodellen inn i en store, som tar seg av håndtering av data som backend ikke gjorde
3. Problemet løses i komponentene ved å håndtere data slik store eller backend ikke gjorde.

Bonus no. 4, er å lage enda en backend for å fikse på den andre backenden. Da har man en dårlig dag.

Stemmer for 3 i dag og 2 i morgen, 1 der det lar seg gjørde (som åpenbart ikke er her)